### PR TITLE
Add PLEXOS documentation to gh-pages

### PR DESCRIPTION
**Pull request by k1nshuk**
PLEXOS documentation currently does not exist on our GitHub pages account. This PR should fix that.

### CODE REVIEW

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+```bash
+$PLEXOS/PLEXOS64 -n 5_bus_system_v2.xml -m 2024_yr_15percPV_MT_Gurobi
+```
+
+The command above assumes that we are running the model `2024_yr_15percPV_MT_Gurobi` from file `5_bus_system_v2.xml`. PLEXOS 9.0RX requires validating user-credentials for a local 
+PLEXOS account for each run. Therefore, if we ran the above command in an interactive session, we would need to enter the following username and password
+
+```txt
+username : nrelplexos
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md:38]: Is it ok to post the username/password publicly, or is that something we give out to specific users/groups?

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+Fortunately, we can bypass the prompt for a local PLEXOS account username and password (useful for slurm batch jobs) by passing them as command line arguments as follows.
+
+```bash
+$PLEXOS/PLEXOS64 -n 5_bus_system_v2.xml -m 2024_yr_15percPV_MT_Gurobi -cu nrelplexos -cp Nr3lplex0s
+```
+
+!!! warning
+    Not providing the username and password in batch jobs **WILL** cause your jobs to fail.
+
+## Example scripts
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md:51]:
```suggestion
## Example Scripts
```

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+The example scripts are available [here](). Please clone the repository to run those examples.
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md]: Needs link

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+### 1: Basic Functionality Test
+
+The basic functionality test is the same the example run in the section above. We will
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md:70]:
```suggestion
The basic functionality test is the same as the example run in the section above. We will
```

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    module load gurobi/10.0.2
+    module load plexos/9.200R06
+
+    # Finally run the PLEXOS executable
+    $PLEXOS/PLEXOS64 -n 5_bus_system_v2.xml -m 2024_yr_15percPV_MT_Gurobi -cu nrelplexos -cp Nr3lplex0s
+    ```
+
+### 2: Simple batch script submission
+
+We will run the same example by submitting the job to the SLURM queue so. In order to run this example as is, run the following commands
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md:95]:
```suggestion
We will run the same example by submitting the job to the SLURM queue. In order to run this example as is, run the following commands
```

File: docs/Documentation/Applications/Plexos/run_plexos.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    git clone git@github.com:NREL/HPC.git
+
+    # Go to the appropriate folder and submit the job on the HPC
+    cd HPC/applications/plexos/RunFiles
+    sbatch -A account_name --mail-user=your.email@nrel.gov submit_simple.sh
+    ```
+
+### 3: Enhanced batch script submission
+
+This builds upon the previous example where it tries to run the same model as before, but adds redundancy where the job doesnt fail if a license is not found. The submission script attempts to re-run the job after waiting 120 seconds for each attempt.
```
Comment (yandthj) [inline docs/Documentation/Applications/Plexos/run_plexos.md:114]:
```suggestion
This builds upon the previous example where it tries to run the same model as before, but adds redundancy where the job doesn't fail if a license is not found. The submission script attempts to re-run the job after waiting 120 seconds for each attempt.
```

**Review by yandthj — CHANGES_REQUESTED** [review]

File: docs/Documentation/Applications/Plexos/run_plexos.md (hunk 3 above)
Comment (k1nshuk) [reply docs/Documentation/Applications/Plexos/run_plexos.md]: I am reworking the master branch that will contain the example files, so things Amy unfortunately remain broken until another PR to the master branch gets merged in. Let me set up the master branch PR as a draft in the meantime.

File: docs/Documentation/Applications/Plexos/run_plexos.md (hunk 1 above)
Comment (k1nshuk) [reply docs/Documentation/Applications/Plexos/run_plexos.md:38]: I think it's okay to have this password available in the open as the only thing it enables is for someone to run PLEXOS, assuming that they are already on the network and can ping the license server.

**Review by harrysorensennrel — DISMISSED** [review]
There are ways to generate inputs that do not require the windows GUI using spreadsheets and/or scripts, but it probably doesn't belong in this documentation.

File: docs/Documentation/Applications/Plexos/run_plexos.md (hunk 3 above)
Comment (yandthj) [reply docs/Documentation/Applications/Plexos/run_plexos.md]: Merged in the master branch PR, so link can be updated now.

**Comment by yandthj** [comment]
Merged in master branch PR, so can update links in this PR now. Thanks!

**Comment by k1nshuk** [comment]
I just added the links. Can be merged now.

**Review by yandthj — APPROVED** [review]

**Pull request merged**